templates/measure-matrix/main.go

```diff
@@ -109,6 +109,13 @@ func main() {
 
 	// Write the output.
 	file, _ := json.MarshalIndent(out, "", " ")
+
+	// [gosec issue] G306 (CWE-276): Expect WriteFile permissions to be 0600 or less
+	// macOS default file permission is 0644 -rw-r--r--
+	// Linux default file permission is 0660 -rw-rw-r--
+	// This file we created contains no secret or credential. 0644 permission should not have any security risk.
+	// If a user adds sensitive content in this file later, it is the user's responsibility to protect it.
+	/* #nosec */
```

Comment:
This file is part of a template that will be user facing (generate via: `nextmv sdk init -t measure-matrix`). I think we should avoid having such internal comments in there. Maybe we can add the template dir to the exclusions?

Reply:
I am removing most of the comment but leaving `#nosec`. For now I am a little hesitated to exclude the entire `templates/` dir from gosec scan. After all, we are responsible to provide good, secure template code samples for users to get started safely.